task1/src/app/cource-item.ts

```diff
@@ -6,9 +6,10 @@ export class CourceItem implements ICourceItem {
     ConductAt:Date; 
     DurationMin:number;
     Description:string;
+    Starred:boolean;
 
 
-    constructor(id:number = null, title: string  = null, conductAt:string  = null, durationMin:number  = null, description:string  = null) {
+    constructor(id:number = null, title: string  = null, conductAt:string  = null, durationMin:number  = null, description:string  = null, starred:boolean = false) {
```

Comment:
chodziło mi o trochę inną implementację, jako argument wejściowy jest jeden obiekt json, który ma odpowiednie atrybuty
```code
constructor (atrributes: ICourceItem) {
    Object.assign(this, attributes)
    this.ConductAt = new Date(    this.ConductAt );
```

no ale to teraz wymagałoby lekkich przeróbek, zastanów się nad tym i jak coś to możemy to obgadać